### DISCUSSION TITLE
Is it possible to configure options.Scheduling.IgnoreDuplicates via Classic property-based configuration?

### DISCUSSION THREAD

**Discussion by trtrkl** [quartznet/quartznet#2306] [Q&A] [answered]
Currently, my program use Classic property-based for configuration. However, I come across these two properties in the example.

![image](https://github.com/quartznet/quartznet/assets/27114339/ed20882f-1a22-4920-8504-e457450cf3e1)

and doesn't seem to be in the [Configuration Reference]( https://www.quartz-scheduler.net/documentation/quartz-3.x/configuration/reference.html). My question is can I set it in the property-based configuration like this?

`var props = new NameValueCollection
{
    ["quartz.scheduling.IgnoreDuplicates"] = "true",
};`

**Comment by lahma** [comment, marked as answer]
Currently it's not possible as the value of the behavior is either read from XML scheduling or DI container configuration syntax state (in-memory).

**Reply by trtrkl** [reply]
I see. Thank you 😄 🙏
